roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml

```diff
@@ -40,12 +40,19 @@
   - hawkular-metrics
   - hawkular-jgroups-keystore
 
-- name: generate htpasswd file for hawkular metrics
+- name: generate htpasswd content for hawkular metrics
   shell: >
-    docker run -i -a STDIN -a STDOUT -v "{{ mktemp.stdout }}"
+    cat "{{ mktemp.stdout}}/hawkular-metrics.pwd" | cat > metrics.pwd && docker run -i -a STDIN -a STDOUT
     "{{ openshift_metrics_httpd_image }}" htpasswd -ni hawkular
-    > '{{ mktemp.stdout }}/hawkular-metrics.htpasswd'
-    < '{{ mktemp.stdout }}/hawkular-metrics.pwd'
+    < metrics.pwd
```

Comment:
yeah, i must have been testing it incorrectly. it validated using the approach you pasted above